src/routes/homepage/sections/howDoesTLIPWork.svelte

```diff
@@ -1,8 +1,118 @@
+        title: "How Does TLIP Work?"
+    };
+    const CARDS = [
+        {
+            title: "Decentralised",
+            description: "Data is not stored or controlled centrally.",
+            icon: {
+                name: "decentralized",
+                fill: "#00874C"
+            }
+        },
+        {
+            title: "Secure",
+            description: "State-of-the-art security encryption avoids hacking by unauthorised parties.",
+            icon: {
+                name: "secure",
+                fill: "#00874C"
+            }
+        },
+        {
+            title: "Upgrades Legacy IT",
+            description: "Plugs into existing systems and facilitate expansion through open APIs.",
+            icon: {
+                name: "upgrades",
+                fill: "#00874C"
+            }
+        },
+        {
+            title: "Data at the Source",
+            description: "Keeps data local and traceable to source.",
+            icon: {
+                name: "data",
+                fill: "#00874C"
+            }
+        },
+        {
+            title: "Data Owners Control Access",
+            description:
+                "Gives data producers control over who can access their data, thus guaranteeing privacy and confidentiality.",
+            icon: {
+                name: "dataAccess",
+                fill: "#00874C"
+            }
+        },
+        {
+            title: "Innovative",
+            description: "Immutable transparency and auditability through distributed ledger technology.",
+            icon: {
+                name: "innovative",
+                fill: "#00874C"
+            }
+        }
+    ];
+    const SEE_IN_ACTION = {
+        title: "See in Action",
+        cards: [
+            {
+                video: "/assets/how-it-works-video.png",
+                title: "Demo Video Title",
+                description: "Interoperable. No vendor-lock in. Working with legacy systems."
+            },
+            {
+                video: "/assets/how-it-works-video.png",
+                title: "Flowers from Kenya to Holland",
+                description:
+                    "The benefits of TLIP are demonstrated by a shipment of flowers from Nairobi, Kenya, to Royal FloraHolland in the Netherlands."
+            }
+        ]
+    };
 </script>
 
-<section class="container py-72" {id}>
-    <div>How does TLIP</div>
-    <hr />
+<section {id}>
+    <div class="container pt-24 lg:pt-40 pb-16 lg:pb-32">
+        <h2 class="mb-12 lg:mb-16 text-center ">{HEADINGS.title}</h2>
+        <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-10 mb-32">
+            {#each CARDS as { title, description, icon }}
+                <Card classes="work bg-white" withBorder>
+                    <div slot="eyebrow" class="mb-6">
+                        <Icon {...icon} />
+                    </div>
+                    <h3 slot="title" class="tracking-0.02 mb-2 leading-110">{title}</h3>
```

Comment:
This looks more like `mb-4`

Reply:
Looks, but is 8px :wink: 
![Screenshot from 2022-06-13 12-43-28](https://user-images.githubusercontent.com/72078237/173337133-dee532de-7510-4cdd-8892-35a891d093cb.png)

Reply:
But that's because of the boxes, the actual text is 16px apart 
![image](https://user-images.githubusercontent.com/88061365/173338201-f9c17c90-1716-43a5-97cf-d201cbcedd6e.png)